suite-native/app/e2e/tests/entropyCheckFailT3T1.test.ts

```diff
@@ -0,0 +1,68 @@
+import { conditionalDescribe } from '@suite-common/test-utils';
+import { DeviceReducerState, deviceReducerInitialState } from '@suite-common/wallet-core';
+
+import { deviceChecksEnabledState } from '../fixtures/deviceChecksEnabledState';
+import { onboardingCompletedState } from '../fixtures/onboardingCompletedState';
+import { onDeviceOnboarding } from '../pageObjects/deviceOnboardingActions';
+import { openApp, preparePreloadedReduxState, prepareTrezorEmulator } from '../support/setup';
+import { wait, waitForVisible } from '../support/utils';
+
+const getPreloadedState = (payload: DeviceReducerState['simulatedEntropyCheckFail']) =>
+    preparePreloadedReduxState(onboardingCompletedState, deviceChecksEnabledState, {
+        device: { ...deviceReducerInitialState, simulatedEntropyCheckFail: payload },
+    });
+
+const LONG_RUNNING_TEST_TIMEOUT = 5 * 60 * 1000; // [ms]
+
+conditionalDescribe(
+    device.getPlatform() === 'android',
+    'Simulated entropy check failure on T3T1 [@specificModel]',
+    () => {
+        beforeEach(async () => {
+            await prepareTrezorEmulator({ model: 'T3T1', seed: '' });
+        });
+
+        it(
+            'Device compromised (entropy-mismatch)',
+            async () => {
+                await openApp({
+                    args: {
+                        preloadedState: getPreloadedState({
+                            success: false,
+                            payload: { code: 'Failure_EntropyCheck', error: 'SIMULATED ERROR' },
+                        }),
+                    },
+                });
+                await onDeviceOnboarding.proceedToCreateOrRecoverCrossroads();
+                await onDeviceOnboarding.startCreatingWallet();
+
+                await onDeviceOnboarding.waitForDeviceCompromisedModal();
+            },
+            LONG_RUNNING_TEST_TIMEOUT,
+        );
+
+        it(
+            'Transport error (device disconnected during entropy check)',
+            async () => {
+                await openApp({
+                    args: {
+                        preloadedState: getPreloadedState({
+                            success: false,
+                            payload: {
+                                code: 'Failure_EntropyCheck',
+                                error: 'device disconnected during action',
+                            },
+                        }),
+                    },
+                });
+                await onDeviceOnboarding.proceedToCreateOrRecoverCrossroads();
+                await onDeviceOnboarding.startCreatingWallet();
+
+                await waitForVisible(by.id('@toast'));
```

Comment:
IMHO, would be good to assert the toast text
```suggestion
                await waitForVisible(by.id('@toast'));
                await detoxExpect(element(by.id('@toast'))).toHaveText('device disconnected during action')

Reply:
Done :heavy_check_mark: